### ISSUE TITLE
"List all versions" no longer working

### ISSUE THREAD

**Issue by TomazErjavec** [clarinsi/clarin-dspace#34]
With entries that have more than one version, the landing page has an "Other versions" field, where you have a drop down menu "List all version", which should show you all the versions of the resource, cf. e.g. http://hdl.handle.net/11356/1210

With (probably) the latest commit, this drop-down menu no longer works, it shows the spinning wheel forever. Tested on Firefox and Chrone.

**Comment by cyplas** [comment]
Well well, this is a mysterious one. After some investigation, including being puzzled by this only occurring on fido, I decided to try reverting to the previous commits on fido in the evening, despite being skeptical that it would help. But it did help. But then it also worked fine when I restored the commits right back ... The only change I made in between was to do the routine weekly ubuntu system updates and reboot fido. These updates did include apache and java, hmm ...

Well, closing this, though it wouldn't hurt, @TomazErjavec, if you also check that it works for you now.
